bridge_adaptivity/bridge_lti/urls.py

```diff
@@ -4,6 +4,6 @@
 from .consumer import source_preview
 
 urlpatterns = [
-    url(r'^launch/(?P<collection_id>\d*)$', lti_launch, name='launch'),
+    url(r'^launch(?P<collection_id>\d*)$', lti_launch, name='launch'),
```

Comment:
That is not correct, slash is required here, please revert.

Reply:
But it doesn't catch the `/lti/launch` case.

Reply:
Why we need to care about '/lti/launch' when we have `/lti/launch/`?

Reply:
Because of 50% of cases, instructors will mistyped it?

Reply:
That is not the case, django appends the slash to the end of the URL by default.